crates/core/src/bc/xml.rs

```diff
@@ -1566,6 +1569,48 @@ pub struct Schedule {
     pub time_block_list: TimeBlockList,
 }
 
+/// List of users
+#[derive(PartialEq, Eq, Default, Debug, Deserialize, Serialize)]
+pub struct UserList {
+    /// XML Version
+    #[serde(rename = "@version")]
+    pub version: String,
+    /// The actual user-list
+    #[serde(rename = "User", skip_serializing_if = "Option::is_none")]
+    pub user_list: Option<Vec<User>>,
+}
+
+/// A struct for reading and writing camera user records
+#[derive(PartialEq, Eq, Default, Debug, Deserialize, Serialize)]
+pub struct User {
+    /// The user_name is used to identify the user in the API
+    #[serde(rename = "userName")]
+    pub user_name: String,
+    /// The password seems to only be included when creating or modifying a user
+    #[serde(rename = "password", skip_serializing_if = "Option::is_none", skip_deserializing)]
```

Comment:
```suggestion
    #[serde(
        rename = "password",
        skip_serializing_if = "Option::is_none",
        skip_deserializing
    )]
```

Rust fmt... Should do this on my mac rather than over github